docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1
+
+#
+# INSTALL MAVEN DEPENDENCIES
+#
+FROM maven:3.8.5-openjdk-18 AS builder
+
+# MAVEN: BOM
+FROM builder as bom-stage
+
+ARG DOCKER_LDES_BOM_REPO
+RUN git clone ${DOCKER_LDES_BOM_REPO} bom
+
+# MAVEN: application
+FROM builder as app-stage
+
+COPY --from=bom-stage bom/pom.xml .
+RUN mvn install
+WORKDIR /
+COPY . /
+RUN mvn install
```

Comment:
This is incorrect. Since you are in the folder `docker`, you copy the contents of the folder `docker`, which is the `Dockerfile`. To solve this issue you could either clone the REPO or move de Dockerfile one level higher (this is needed as the Dockerfile can only COPY on its own level and below).
There's also no distinction between line 18 and line 21. These install the bom twice.

Reply:
The context is derived from the context declaration in docker-compose.yml.
The first mvn install installs the BOM in the local repo, the COPY makes sures that the second mvn install installs the app

Reply:
Aha, indeed. Tried building the container in the `docker` folder. In that sense, it would be safer to clone the repo, but can agree with this as well.